KlingonAssistant/data/mem-07-l.xml

```diff
@@ -3322,7 +3322,7 @@ This prefix is sometimes dropped for unknown reasons. While an error, this is of
       <column name="entry_name">lupDujHomwIj lubuy'moH gharghmey.</column>
       <column name="part_of_speech">sen:phr</column>
       <column name="definition">My hovercraft is full of eels.</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">Mein Boot is voller Würmer.</column>
```

Comment:
The original expression (from Monty Python) specifically refers to a "hovercraft". I think "Boot" might be too generic. Is "Luftkissenfahrzeug" the right term?